sites/cpgnext.com/server/styles/index.scss

```diff
@@ -65,8 +65,12 @@ $theme-site-navbar-secondary-font-size: 16px !default;
 }
 
 .site-newsletter-menu {
-  &__container {
-    top: calculate-navbar-height-for(default, $additional-padding: 40px);
+  &--open {
+    margin-top: calculate-navbar-height-for(default, $additional-padding: 40px);
```

Comment:
@Shinsina there will have to be a `@include media-breakpoint-down(md${1100px}) {}` with no $additional-padding: applied.  looks like the navigation hides at 1100px.  The same for line 73